integration/kubernetes/README.md

```diff
@@ -0,0 +1,299 @@
+## Hazelcast Jet Kubernetes Code Sample
+
+This sample is a guideline on how to start a Hazelcast Jet cluster, submit a Hazelcast Jet job and inspect the cluster via Hazelcast Jet Management Center on the Kubernetes environment.
+
+## Prerequisites
+
+1. Up and running [Kubernetes](https://kubernetes.io) version 1.9 or higher.
+   * For development and testing, you may use [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/)
+   * You must have the Kubernetes command line tool, [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/),
+    installed
+2. Familiarity with `kubectl`, Kubernetes, and Docker.
+3. Git
+4. Apache Maven
+5. Java 8+
+
+
+## Deployment Steps
+
+Starting a Hazelcast Jet cluster consists of a few steps: 
+
+- Creating Role Binding
+- Creating Config Map
+- Starting Hazelcast Jet Cluster
+
+
+### Creating Role Binding
+
+Hazelcast Jet uses Kubernetes API to discover nodes and that is why you need to grant certain permissions. The simplest Role Binding file can look as `rbac.yaml`. Note that you can make it more specific, since Hazelcast actually uses only certain API endpoints. Note also that if you use "DNS Lookup Discovery" instead of "REST API Discovery", then you can skip the Role Binding step at all. Read more at [Hazelcast Kubernetes API Plugin](https://github.com/hazelcast/hazelcast-kubernetes).
```

Comment:
missing new lines. Text files should be wrapped at 72 chars. Goes for whole file.